src/shared/containers/Refile/ClearItemStatusForm.jsx

```diff
@@ -86,44 +65,48 @@ class ClearItemStatusForm extends Component {
   * @param {object} state - the current state object
   */
   validateField(fieldName, state) {
-    if (state.formFields[fieldName].length > 20) {
-      this.setFieldError(fieldName, state, 'The barcode must be no longer than 20 digits');
+    const inputBarcode = state.formFields[fieldName];
+
+    if (inputBarcode.length > 20 || !inputBarcode) {
```

Comment:
Maybe `inputBarcode.length > MAX_BARCODE_LENGTH` would make sense here.

Reply:
We can use a variable to replace the number, however, for refiling, Kate has a different request to check barcode length than other functions. So this variable will be only for refile.

Reply:
👍